example/zfs-over-legacy.nix

```diff
@@ -34,11 +49,16 @@
   zpool = {
     zroot = {
       type = "zpool";
+      rootFsOptions.mountpoint = "none";
       datasets = {
-        zfs_fs = {
+        "root" = {
+          zfs_type = "filesystem";
+          options.mountpoint = "none";
+        };
+        "root/zfs_fs" = {
           zfs_type = "filesystem";
           mountpoint = "/zfs_fs";
-          options."com.sun:auto-snapshot" = "true";
```

Comment:
You don't want to keep autosnapshots here for testing zfs options?

Reply:
was removed for debugging, will return soon (is already in my dirt)